apps/e2e/forms-storybook-e2e/src/e2e/checkbox.component.cy.ts

```diff
@@ -15,6 +15,11 @@ describe('forms-storybook - checkbox', () => {
           .should('be.visible')
           .get('#touched-required-checkbox')
           .dblclick();
+        cy.get('sky-help-inline')
+          .eq(1)
+          .should('exist')
+          .should('be.visible')
+          .click();
```

Comment:
The popover seems to be in an invalid location on the screenshot? 
![Screenshot 2024-04-19 at 12 00 13 PM](https://github.com/blackbaud/skyux/assets/12497062/745dfd1c-ae4d-4ff5-aac7-69c0b60894e7)

Reply:
ugh ok so this seems to be being caused by the screenshot needing to scroll to take the screenshot. It's working when I delete a bunch of other checkboxes to make the screenshot smaller. I'll have to ping john about this next week

Reply:
actually I think im going to remove the checkbox with open help inline, changes to that functionality should not affect checkbox visual test.